src/chrome/chromeUtils.ts

```diff
@@ -332,3 +335,48 @@ export function getUrlRegexForBreakOnLoad(url: IResourceIdentifier): string {
     const escapedFileName = pathToRegex(fileNameWithoutExtension);
     return '.*[\\\\\\/]' + escapedFileName + '([^A-z^0-9].*)?$';
 }
+
+/**
+ * Checks if a given tcp port is currently in use (more accurately, is there a server socket accepting connections on that port)
+ * @param port The port to check
+ * @param host Optional host, defaults to 127.0.0.1
+ * @param timeout Timeout for the socket connect attempt
+ * @returns True if a server socket is listening on the given port, false otherwise
+ */
+export async function isPortInUse(port: number, host = '127.0.0.1', timeout = 400): Promise<boolean> {
```

Comment:
isPortInUse is independent on chrome. I'd move it to another file. portUtilities.ts?